src/main/resources/templates/assign-book.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:th="http://www.thymeleaf.org">
+<head>
+    <meta charset="utf-8"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.1.3/dist/css/bootstrap.min.css"
+          integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO" crossorigin="anonymous">
+
+    <title>Check Out Book</title>
+</head>
+<body>
+<div align="center">
+    <h1>Check Out Book</h1>
+    <a th:href="@{/}">Customer List</a>
+    <br/>
+    <form action="#" th:action="@{/book/assign}" method="post">
+        <input type="hidden" name="customerId" th:value="${customer.id}"/>
+        <table border="0" cellpadding="10" class="table">
+            <tr>
+                <td>Customer Name: <strong><span th:text="${customer.fullName}"></span></strong></td>
+            </tr>
+            <tr>
+                <td>Customer Email: <strong><span th:text="${customer.emailAddress}"></span></strong></td>
+            </tr>
+            <tr>
+                <td>Check out book:
+                    <select class="form-select" th:name="bookId">
+                        <option th:if="${books.isEmpty()}">None Available</option>
+                        <option th:each="book : ${books}" th:value="${book.id}" th:text="${book.toString()}"></option>
+                    </select></td>
+            </tr>
+            <tr>
+                <td>
+                    <button type="submit" th:disabled="${books.isEmpty()}">Save</button>
+                </td>
+            </tr>
+        </table>
+    </form>
+</div>
+</body>
+</html>
```

Comment:
new line

Reply:
new line? I should add a new line somewhere?